### DISCUSSION TITLE
Problems connecting to Faikin module with Android mobile browser after recent FW update

### DISCUSSION THREAD

**Discussion by wurmtal868** [revk/xESP32-Faikout#229] [Q&A] [answered]
As I wrote in another thread I'm testing controlling my multi split units by adjusting the "demand" of the outdoor unit for better power efficiency.
Additionally I disconnect my units from mains over night to eliminate the standby power consumption.
I just realize that power cycling the Faikin could trigger an OTA. 

Now to problem:
2 of my 6 Faikins are not accessible using an Android phone after today's OTA 
![image](https://github.com/revk/ESP32-Faikin/assets/32965910/64dcc615-48bd-42c4-9cf4-d42c4b7e9562)
![image](https://github.com/revk/ESP32-Faikin/assets/32965910/1c77ff4b-7713-45ba-a4d6-083ca4891fb0)

It seems that the mobile browser does not allow unsecured http to the two affected devices, while for the other modules a warning is shown that the connection is not secure.
Same happened on a Linux Mint laptop when connecting my network through WiFi. 
Connecting via cable works fine. 
@revk Do you have any idea what could have gone wrong with the 2 modules?
Since 4 of 6 modules are working fine, I don't think that it is a general FW issue.

**Comment by revk** [comment]
OK they do not do https server.

The OTA has not changed that, and I have not issued a non-beta release for a while.

Surely you must have a way to access that is plain http?

**Reply by wurmtal868** [reply]
Yes, I have, when I use a browser that is connected with a LAN cable to my router.
I actually plugged a USB-LAN dongle into my phone and then it's working with phone as well.
I don't understand why this is only happening on two modules out off 6. the S1-Pico above and and a S3-Mini .
![image](https://github.com/revk/ESP32-Faikin/assets/32965910/8ddffd69-a5a5-4794-a349-0c73f5ee76d0)
When sitting on the couch I usually have no cable connected to my phone.

**Reply by wurmtal868** [reply, marked as answer]
Never mind. 
The issue seems to be caused by my WiFi-Mesh. Depending where my phone is connected either my DSL router directly or via a repeater my phone will or will not connect to the 2 modules in question.
